mobile/src/home-view.js

```diff
@@ -120,10 +288,47 @@ class HomeView extends PureComponent {
       <Image source={trophy} style={s.trophy} />
       <Text style={s.whiteTitle}>{t('challengeCap')}</Text>
       <Text style={s.joinSessionName}>{session.name}</Text>
-      <Button title="Let's Play!" onPress={this.join} />
+      <Button title={t('letsPlay')} onPress={this.join} />
     </View>
   )
 
+  renderAdminSessions = (sessions, me) => {
```

Comment:
HomeView is getting rather large and doing lots of things.  Don't fix in this PR, but we can talk about how to break this up and encapsulate some complexity.